docs/java-rest/high-level/getting-started.asciidoc

```diff
@@ -154,3 +154,23 @@ executes the request. For example, this is the place where you'd specify a
 `NodeSelector` to control which node receives the request. See the
 <<java-rest-low-usage-request-options,low level client documentation>> for
 more examples of customizing the options.
+=== Asynchronous usage
+
+All of the the methods accross the different clients exist in a traditional synchronous and 
+asynchronous variant. The difference is that the asynchronous ones use asynchrous apache 
+httpclient requests. This is useful if you are doing multiple requests or are using e.g.
```

Comment:
can we refer to the low-level client method instead of apache http client?

Reply:
I fixed that now